requirements-dev-sqlalchemy.txt

```diff
@@ -22,3 +22,4 @@ snowflake-sqlalchemy>=1.2.3  # sqlalchemy_tests
 # NOTE - 20210421 sqlalchemy pinned temporarily due to 1.4.10 breaking mssql tests
 sqlalchemy>=1.3.16,<1.4.10 # sqlalchemy_tests
 sqlalchemy-redshift>=0.7.7  # sqlalchemy_tests
+teradatasqlalchemy==17.0.0.1 # sqlalchemy_tests
```

Comment:
It looks like there haven't yet been any tests added here for Teradata. Is this dependency needed at this stage?

Reply:
Yes, teradatasqlalchemy dependency is required for the dialect specific changes. @talagluck Shall I remove the **sqlalchemy_tests** comment part from this?